build_library/build_image_util.sh

```diff
@@ -273,6 +273,58 @@ write_packages() {
     image_packages "$1" | sort > "$2"
 }
 
+get_metadata() {
```

Comment:
Can you please add a few comments regarding what this function does and how it should be invoked?